src/components/data/data-utils.js

```diff
@@ -956,7 +956,11 @@ export const MultipleEntityEditor = ({ entityType, entities, attributeNames, ent
     [!_.includes(attributeToEdit, attributeNames), () => 'The selected attribute does not exist.']
   )
 
-  const [operation, setOperation] = useState('setValue')
+  const Operation = {
```

Comment:
Thanks for introducing this.

For consistency with the rest of the codebase, I'd lower-case the initial in `Operation`, and I think we typically pluralize them.